akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -162,23 +165,16 @@ abstract class ActorMaterializer extends Materializer {
    */
   def isShutdown: Boolean
 
-  /**
-   * INTERNAL API: this might become public later
```

Comment:
I am not sure about removing these markers. I don't feel that we are currently ready to make these APIs officially open. It makes sense to open up to those whom we work closely with (and hence can easily coordinate with), and also evolve it if necessary until we feel comfortable to fully commit to these APIs to the larger public.